tests/test_lambda_potential.py

```diff
@@ -28,6 +29,7 @@ def lambda_potential(conf, params, box, lamb, multiplier, offset, u_fn):
 
 class TestLambdaPotential(GradientTest):
 
+    @unittest.skip("not supported")
```

Comment:
Are unittest markers preferred over the pytest markers?

Reply:
I'm actually not sure, I think unittest markers always get picked up by pytest runners, but pytest markers will not get picked up by unittest runners